setup.py

```diff
@@ -23,7 +22,17 @@
     url='https://github.com/JonathanShor/DoubletDetection',
     author='Adam Gayoso, Jonathan Shor, Ambrose J. Carr',
     author_email='ajg2188@columbia.edu',
-    package_dir={'': 'src'},
```

Comment:
Rationale for the move from `src`?

Reply:
It seems pretty standard to not have an empty src folder just hanging out. Most projects use this arrangement.

For example - https://github.com/numpy/numpy, or pytorch, or Scanpy, etc.